samples/extensions/TIBCO/function/condition/if.go

```diff
@@ -0,0 +1,37 @@
+package condition
+
+import (
+	"fmt"
+	"github.com/project-flogo/core/data"
+	"github.com/project-flogo/core/data/coerce"
+	"github.com/project-flogo/core/data/expression/function"
+)
+
+func init() {
+	_ = function.Register(&fnIf{})
+}
+
+type fnIf struct {
+}
+
+func (fnIf) Name() string {
+	return "if"
+}
+
+func (fnIf) Sig() (paramTypes []data.Type, isVariadic bool) {
+	return []data.Type{data.TypeInt, data.TypeString, data.TypeString}, false
```

Comment:
it should be data.TypeBool, data.TypeAny and data.TypeAny

Reply:
Ok